calico_containers/calico_ctl/node.py

```diff
@@ -328,11 +327,11 @@ def node_bgppeer_add(ip, version, as_num):
     Add a new BGP peer with the supplied IP address and AS Number to this node.
 
     :param ip: The address to add
-    :param version: v4 or v6
+    :param version: 4 or 6
     :param as_num: The peer AS Number.
     :return: None
     """
-    address = check_ip_version(ip, version, IPAddress)
+    address = IPAddress(ip)
     peer = BGPPeer(address, as_num)
     client.add_bgp_peer(version, peer, hostname=hostname)
```

Comment:
These functions will actually have to be updated also.  `datestore.py` contains a lot of references to "v4"/"v6".  These should be changed too.  This might be my fault, I don't think I was very clear on this.